.vscode/scripts/runtime/k3d/install_tooling.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if ! command -v k9s &> /dev/null
+then
+    curl -sS https://webinstall.dev/k9s | bash
```

Comment:
sending result from curl into bash directly is bad security practice, especially if it is performed by sudo user (jfyi)

Reply:
Script is in same state as python. Let's keep them in sync and come up with a solution for all repos.